api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -214,7 +210,7 @@ public void deleteWorkspace(DbWorkspace dbWorkspace) {
     fireCloudService.deleteWorkspace(
         dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getFirecloudName());
     dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED);
-    dbWorkspace = workspaceDao.saveWithLastModified(dbWorkspace);
+    dbWorkspace = workspaceDao.saveWithLastModified(dbWorkspace, userProvider.get());
```

Comment:
ignoring Codacy because `dbFoo = dao.save(dbFoo)` is a pattern we need to use for Hibernate to work correctly.